tox.ini

```diff
@@ -1,20 +1,19 @@
 [tox]
-skipsdist = True
 envlist = py37, py38, py39, lint
 
 [testenv:lint]
 deps =
     .[test]
 commands =
-    flake8 aicsimageio --count --verbose --show-source --statistics
-    isort aicsimageio --check-only
-    mypy aicsimageio
-    black --check aicsimageio
+    flake8 aicsimageio --count --verbose --show-source --statistics --exclude aicsimageio/metadata/czi-to-ome-xslt/
+    isort aicsimageio --check-only --skip aicsimageio/metadata/czi-to-ome-xslt/
+    mypy aicsimageio --exclude aicsimageio/metadata/czi-to-ome-xslt/
+    black --check aicsimageio --exclude aicsimageio/metadata/czi-to-ome-xslt/
 
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
 deps =
     .[test]
 commands =
-    pytest --basetemp={envtmpdir} --cov-report html --cov=aicsimageio aicsimageio/tests/ {posargs}
+    pytest --basetemp={envtmpdir} --cov-report xml --cov=aicsimageio aicsimageio/tests/ {posargs}
```

Comment:
Why the change here?

Reply:
I was matching the `pytest` command that was formerly in the workflow files., which was this:
```
- name: Test with pytest
  run: |
    pytest --cov-report xml --cov=aicsimageio aicsimageio/tests/
```
I'm not sure if XML or HTML is preferable here, but I just didn't want to risk breaking the workflow.

Reply:
Ahhh I see. XML is used for the codecov bot while HTML is used for rendering the coverage locally. If you are using firefox: `firefox htmlcov/index.html` will open them up for example. But it seems we can generate multiple by just adding them all: https://stackoverflow.com/a/66280712

Can you change this to have both `--cov-report xml --cov-report html`

Reply:
Will do!